addons/mail/static/src/model/tests/test_models.js

```diff
@@ -0,0 +1,89 @@
+/** @odoo-module **/
+
+import { registerNewModel } from 'mail/static/src/model/model_core.js';
+import { attr, many2many, many2one, one2many, one2one } from 'mail/static/src/model/model_field.js';
+
+function factoryContact(dependencies) {
+    class Contact extends dependencies['mail.model'] {
+        static _createRecordLocalId(data) {
+            if (data.id) {
+                return `${this.modelName}_${data.id}`;
+            }
+            else {
+                return _.uniqueId(`${this.modelName}_`);
+            }
+        }
```

Comment:
But do you actually need a local id for those? Shouldn't be needed if you remove `findFromIdentifyData` from tests

Reply:
yes, it is needed because we need to get the records created in nested `create` or `insert` commands